tasks/coffee-shop-layout/coffee-shop-layout.md

```diff
@@ -0,0 +1,207 @@
+# Coffee House
+
+## Project's Description
+The Coffee House project involves creating a responsive website with two pages, incorporating interactivity to enhance user engagement. The project challenges you to demonstrate your front-end development skills while adhering to responsive design principles.
+The duration of the task is 2 week.  
+The form of evaluation of the task is a cross-check review.
+[Design in Figma](https://www.figma.com/file/SAoBmuOqTfguehdT4IFRxQ/Coffee-House?type=design&node-id=0-1&mode=design&t=qis81E9Ovgx47eVl-0)
+
+
+## Key skills
+- Valid semantic responsive web design;
+- Easy-to-maintain readable code;
+- Exporting styles and graphics from Figma;
+- Using JavaScript to implement the functionality specified in the task.
+
+
+## Creating your own copy of the layout
+Start the task by creating your own copy of the layout. To do this:
+- Log in to [Figma](https://www.figma.com/);
+- Open the layout;
+- On the top panel, click on the arrow next to the layout name, and select the option "Duplicate to your drafts";
+- At the top left, open the settings, and choose "Back to files";
+- Open the copy of the layout with the label "In Drafts".
+
+
+## Technical requirements
+1. The layout is valid, semantic, and matches the design.
+2. The application is displayed correctly and functions properly in the latest version of Google Chrome.
+3. Using CSS frameworks (e.g., `Bootstrap`) **is not allowed**.
+4. Using JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.) **is not allowed**.
+5. Using outdated libraries (e.g. `JQuery`, etc.) or pre-built libraries (e.g. `Swiper`, etc) to implement functionality **is not allowed**.
+6. Using `TypeScript` **is not allowed**.
+7. Using CSS preprocessors (`SASS`, `SCSS`), `normalize.css` **is allowed**.
+8. Using a style reset with `reset.css` **is not recommended**.
+9. Adding layout as an image by taking a screenshot of a part of the layout and pasting it into the markup **is not allowed**. Please use tags and characters for layout, and use images only for adding pictures and icons, not for layout elements (buttons, blocks, sections).
+10. The code must be readable, without minification or obfuscation.
+
+## Repository Requirements
+- Task should be done in private school's repository

```

Comment:
The task must be completed in your own private repository. Please ensure your repository is set to private to prevent plagiarism or the copying of solutions.